filter/filtering.go

```diff
@@ -0,0 +1,67 @@
+// Package filter provides filter model
+package filter
+
+import (
+	"database/sql"
+
+	"github.com/google/uuid"
+)
+
+type Postgres struct {
+	db *sql.DB
+}
+
+// NewPostgres returns Postgres with db
+func NewPostgres(db *sql.DB) *Postgres {
+	return &Postgres{db: db}
+}
+
+// AddFilter adds new filter to database
+func (p *Postgres) AddFilter(filter *Filter) error {
+	sqlStatement := "INSERT INTO FILTERS(name,type,configutation,reversed,user_id) VALUES ($1,$2,$3,$4,$5)"
+	_, err := p.db.Query(sqlStatement, filter.Name, filter.Type, filter.Configuration, filter.Reversed, filter.UserID)

```

Comment:
Suggest better to use db.Exec for statment that doesn't return rows

Reply:
Yeah, you are wright,  fixed